static/src/javascripts/projects/common/views/svg.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 export const addClassesAndTitle = (
     markup: string,
-    classes?: ?Array<string>,
-    title?: ?string
+    classes?: Array<string>,
```

Comment:
I couldn't find any nullable uses of this, only optional ones, so thought that worth making explicit.